libs/iavl/mutable_tree_oec.go

```diff
@@ -53,8 +53,10 @@ func (tree *MutableTree) SaveVersionAsync(version int64, useDeltas bool) ([]byte
 	if tree.root != nil {
 		if useDeltas {
 			tree.updateBranchWithDelta(tree.root)
-		} else {
+		} else if produceDelta {
```

Comment:
global var?

Reply:
is this from commit 9452c5795128c073ed74c315a2f6b408be0f66d6 (Thu Dec 9 19:47:42)